lib/begin.go

```diff
@@ -196,14 +197,31 @@ func (a *ACBuild) writeEmptyManifest() error {
 		return err
 	}
 
+	archvalue := runtime.GOARCH
+	if runtime.GOOS == "linux" && archvalue == "arm" {
+		elfFile, err := elf.Open("/proc/self/exe")
```

Comment:
.. so why do this rather than my 0x01020304 approach suggested [earlier](https://github.com/appc/acbuild/pull/195#discussion_r58650830)?